js/src/data/test/reducer.test.js

```diff
@@ -418,87 +413,42 @@ describe( 'reducer', () => {
 	} );
 
 	describe( 'Policy Check', () => {
-		it( 'should return with allowed countries', () => {
+		it( 'should return with policy check info', () => {
 			const data = {
-				policy_check: {
-					allowed_countries: {
-						CA: { currency: 'CAD', name: 'Canada' },
-						US: { currency: 'USD', name: 'United States' },
-					},
+				allowed_countries: {
+					CA: { currency: 'CAD', name: 'Canada' },
+					US: { currency: 'USD', name: 'United States' },
 				},
-			};
-			const action = {
-				type: TYPES.RECEIVE_ALLOWED_COUNTRIES,
-				data,
-			};
-			const state = reducer( prepareState(), action );
-
-			state.assertConsistentRef();
-			expect( state ).toHaveProperty(
-				'mc.policy_check.allowed_countries',
-				data.policy_check.allowed_countries
-			);
-		} );
-
-		it( 'should return with store_ssl', () => {
-			const data = {
-				policy_check: {
-					store_ssl: true,
+				store_ssl: true,
+				payment_gateways: {
+					id: 'wc_custom_pg',
+					title: 'Custom Payment Gateway',
+					method_description: 'Description of the payment gateway',
 				},
+				refund_returns: 'Refund and Returns Policy',
 			};
 			const action = {
-				type: TYPES.RECEIVE_IS_STORE_SSL,
+				type: TYPES.POLICY_CHECK,
 				data,
 			};
 			const state = reducer( prepareState(), action );
 
 			state.assertConsistentRef();
 			expect( state ).toHaveProperty(
-				'mc.policy_check.store_ssl',
-				data.policy_check.store_ssl
+				'mc.policy_check.refund_returns',
+				data.refund_returns
```

Comment:
This test is going to pass because we return data in the request and then compare if it matches the same data. But since we changed the code in the PHP it should be looking for a value that's `true` here. Wouldn't it be better to explicitly test for `true` or `false` here? Same for checking all 4 of the fields.

Reply:
Updated.